docs/source/keywords.rst

```diff
@@ -217,10 +221,26 @@ Backend-specific options
     * * ``Kokkos_ENABLE_ATOMICS_BYPASS``
       * Disable atomics when no host parallel nor device backend is enabled for Serial only builds (since Kokkos 4.3)
       * ``OFF``
-       
-
+    * * ``Kokkos_ENABLE_IMPL_CUDA_MALLOC_ASYNC``
+      * Enable ``cudaMallocAsync`` (requires CUDA Toolkit 11.2). This optimization may
+	improve performance in applications with multiple CUDA streams per device, but it
+	is known to be incompatible with MPI distributions built on older versions of UCX
+	and many Cray MPICH instances. See `kokkos/kokkos#7353 <https://github.com/kokkos/kokkos/pull/7353>`_
```

Comment:
Please refer to the CUDA section in the Known Issues page `known-issues.html#cuda` instead of the PR (I don't remember the canonical way to do cross references in reStructuredText on top of my head)

@cedricchevalier19 has been assigned to update that paragraph at the developer meeting yesterday.